events/lastAvatarStandingTrivia/clientScripts/trivia.html

```diff
@@ -0,0 +1,185 @@
+<html>
```

Comment:
Please add `<!DOCTYPE html>` to the top of the file.

Reply:
fixed